docs/qa/test_engineering.rst

```diff
@@ -9,36 +9,39 @@ The Delivery Console is not currently being tested, as it’s a much lower prior
 The test strategy incorporates manual and automated testing. Different approaches are outlined for each area. 
 
 Normandy Server
----------------
+--------------
 The Normandy Server is an API server with recipes. This feature is high priority, and needs to maintain a high level of service. There are two versions with different functionality. V1 is the server clients connect to. V3 is the server that Admins connect to.
 
-- v1
-Automation:
-Test Engineer: chartjes
-Status:  In process
+* v1
+	* Automation:
+	* Test Engineer: chartjes
+	* **Status:**  In process
 
-- v3
-Automation:
-Test Engineer: b4hand
-Manual tests: none
-Status: In process
+* v3
+	* Automation:
+	* Test Engineer: b4hand
+	* Manual tests: none
+	* **Status:** In process
 
 Normandy Firefox client
 -----------------------
-     The Normandy Firefox client queries the Normandy server for instructions and picks up recipe changes. 
-QA: cmuresan
-Automation: 
-Test Engineer: b4hand
-Manual tests: https://testrail.stage.mozaws.net/index.php?/suites/view/232&group_by=cases:section_id&group_order=asc&group_id=16918
-Status: b4hand is reviewing and fixing outdated UI Automation tests. Ciprian is reviewing and updating manual tests.
+The Normandy Firefox client queries the Normandy server for instructions and picks up recipe changes. 
+
+* QA: cmuresan
+* Automation: 
+* Test Engineer: b4hand
+* [Manual tests](https://testrail.stage.mozaws.net/index.php?/suites/view/232&group_by=cases:section_id&group_order=asc&group_id=16918)
```

Comment:
This isn't how RST handles links. Here's an example of how to do an inline link:

https://github.com/mozilla/normandy/blob/003cdfa7621c74630432a0b7134ab818f8ddde74/docs/dev/remote-settings.rst#L56-L58

And here is an example of a not-inline link, where the URL is placed separately to improve code readability.

https://github.com/mozilla/normandy/blob/003cdfa7621c74630432a0b7134ab818f8ddde74/docs/dev/install.rst#L10-L20